modbuslistener.cpp

```diff
@@ -113,6 +112,8 @@ void ModbusListener::on_readButton_clicked()
 
     for ( int slave_id = temperature_sensor_address; slave_id <= electrical_sensor_address; slave_id++ )
     {
+    // QA: is it okay that you create several QModbusDataUnit object? 
+    // QA: do you free memory somewhere? 
```

Comment:
Thank you for this point. It can be very useful. I'll watch it carefully later. Now I'm not sure in this method.